sdk/js/__tests__/Client.spec.js

```diff
@@ -134,6 +135,18 @@ describe('DVCClient tests', () => {
         expect(client.config).toBeUndefined()
     })
 
+    it('should have same value for anonymous user id from local storage and user_id if anonId exists and isAnonymous is true', async () => {
+        const client = new DVCClient('test_env_key', { isAnonymous: true })
+        const anonymousUserId = client.store.load(StoreKey.AnonUser)
+        expect(JSON.parse(anonymousUserId)).toEqual(client.user.user_id)
+    })
+
+    it('should not save anonymous user id in local storage if isAnonymous is false', async () => {
+        const client = new DVCClient('test_env_key', { user_id: 'user1', isAnonymous: false })
+        const anonymousUserId = client.store.load(StoreKey.AnonUser)
+        expect(anonymousUserId).toBeNull()
+    })
+
```

Comment:
we need a test to check an anon_user gets the same id as the one in local storage

Reply:
umm... this test `should have same value for anonymous user id from local storage and user_id if anonId exists and isAnonymous is true` includes the checking "anonymous user id =  the one in local storage". Is there something else I need to add into this?

Reply:
you need a new test, the existing one is checking that the anon user's id was saved to local storage. The new test needs to check that a new anon user's id gets set to the same value as the one saved in local storage